sr_demos/scripts/sr_demos/demo.py

```diff
@@ -174,18 +174,15 @@ def run(self):
             elif input_val == "4":
                 robot.grasp_demo()
             elif input_val == "5":
-                if robot.hand_type == 'hand_e':
-                    robot.random_sequence()
-                else:
-                    rospy.logerr("This demo only works for a 5-fingered Hand E. Please try demos 1-4")
+                robot.random_sequence()
```

Comment:
Shouldn't we have an error here in case someone presses 5 using a hand extra lite? Or maybe this check could be right after reading the input. Like:
```suggestion
                if robot.hand_type == 'hand_extra_lite' and input_val not in ['1','2','4']:
                    rospy.logwarn(f"Not a valid option for {robot.hand_type}")
                    continue

```

Reply:
Slightly different but fixes the issue https://github.com/shadow-robot/sr_interface/pull/825/commits/13cce3a990874f24ff6844070ff5e1bb3eee3ef5